config/jobs/kubernetes-csi/csi-driver-smb/csi-driver-smb-config.yaml

```diff
@@ -375,6 +380,11 @@ presubmits:
         base_ref: release-1.7
         path_alias: sigs.k8s.io/cluster-api-provider-azure
         workdir: true
+      - org: kubernetes-sigs
+        repo: cloud-provider-azure
+        base_ref: release-1.23
```

Comment:
@andyzhangx any reason we're still testing with k8s 1.23 in all these tests? the CCM branch needs to match the k8s release branch for compatibility

Reply:
Is there a legacy reason to prefer k8s 1.23 for a particular scenario @marosset @jsturtevant ?

Reply:
sorry, meant to say "for a particular Windows scenario"

Reply:
I don't think there was a specific reason to keep this stuck at K8s v1.23

Reply:
Not that I know of

Reply:
for csi driver, we would like to run e2e test on different k8s versions to make test coverage as big as possible, let's keep the version as it is now, thanks.